hw/top_earlgrey/cdc/cdc_waivers.spi_device.tcl

```diff
@@ -0,0 +1,25 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Verix CDC waiver file
+# Expression:
+#  ControlSignal==""
+#  ReconSignal==""
+#  MultiClockDomains=="IO_DIV2_CLK::IO_DIV4_CLK"
+
+set_rule_status -rule {W_RECON_GROUPS} -status {Waived}         \
+   -expression {(ControlSignal=~"*u_status_23_to_1_sync.*") &&       \
+                (ReconSignal=~"*u_spid_status.outclk_p2s_byte_*")} \
+   -comment {status bits are converged into SPI line. But transmit bit-by-bit. OK to send old and new status bits in a transaction}
+
+set_rule_status -rule {W_CNTL} -status {Waived}   \
```

Comment:
can this and the next waiver be removed, now that we declared the synchronizer module?

Reply:
Yep. After confirming this works, I will remove.

BTW, this `set_user_xxx` command should run between `elaborate` and `analyze_intent`. So, the module name was uniquified. This one does not work at this time. I am figuring out how to specify.